src/test/java/seedu/address/testutil/pet/TypicalPets.java

```diff
@@ -68,21 +71,27 @@ private TypicalPets() {} // prevents instantiation
      * Returns a {@code PetTracker} with all the typical pets.
      */
     public static PetTracker getTypicalPetTracker() {
-        PetTracker ab = new PetTracker();
+        PetTracker pt = new PetTracker();
         for (Pet pet : getTypicalPets()) {
-            ab.addPet(pet);
+            pt.addPet(pet);
         }
-        return ab;
+
+        return pt;
     }
 
     /**
      * Returns a {@code ModelManager} with all the typical pets.
      */
     public static Model getTypicalModelManager() {
-        return new ModelManager(getTypicalPetTracker(), new UserPrefs());
+        ModelManager modelManager = new ModelManager(getTypicalPetTracker(), new UserPrefs());
+        TypicalSlotsGenerator slotsGen = new TypicalSlotsGenerator(modelManager);
+        for (Slot slot : slotsGen.getTypicalSlots()) {
+            modelManager.addSlot(slot);
+        }
+        return modelManager;
     }
 
     public static List<Pet> getTypicalPets() {
-        return new ArrayList<>(Arrays.asList(AMY, BOB, CARL, DANIEL, ELLE, FIONA, GEORGE));
+        return new ArrayList<>(Arrays.asList(AMY, BOB, CARL, DANIEL, ELLE, FIONA, GEORGE, HOON, IDA, COCO, GARFIELD));
```

Comment:
need to change the value in JsonSerializablePetTrackerTest -> toModelType_typicalPetsFile_success(), from 7 to 11 since 4 more pets are added in here.

Reply:
Also, HOON here is same as the HOON pet in JsonPetTrackerStorageTest . Therefore there is a duplicate pet exception at line 75 in JsonPetTrackerStorageTest . Maybe can leave out HOON here. (the above comment change from 11 to 10)